core/server/config/overrides.json

```diff
@@ -6,6 +6,7 @@
         "helperTemplates": "core/server/helpers/tpl/",
         "adminViews": "core/server/web/admin/views/",
         "defaultViews": "core/server/views/",
+        "defaultRoutes": "core/server/services/route/",
```

Comment:
Not sure, if this is needed, and we'll still change the location for the default routes, but for now I thought it makes sense.